plenum/test/node_request/test_different_ledger_request_interleave.py

```diff
@@ -1,73 +1,111 @@
+                                             sdk_wallet_client, 3))
 
     # Do view change without waiting for replies
     ensure_view_change(looper, nodes=txnPoolNodeSet)
     checkProtocolInstanceSetup(looper, txnPoolNodeSet, retryWait=1)
 
     # Make sure all requests are completed
-    waitForSufficientRepliesForRequests(looper, client1,
-                                        requests=requests)
-
-    ensure_pool_functional(looper, txnPoolNodeSet, wallet1, client1)
-
-    new_steward, new_steward_wallet = addNewSteward(looper, client_tdir,
-                                                    steward1, stewardWallet,
-                                                    'another_ste')
+    total_timeout = sdk_eval_timeout(len(requests), len(txnPoolNodeSet))
+    sdk_replies = sdk_get_replies(looper, requests, timeout=total_timeout)
+    for req_res in sdk_replies:
+        sdk_check_reply(req_res)
+    sdk_ensure_pool_functional(looper, txnPoolNodeSet,
+                               sdk_wallet_client, sdk_pool_handle)
+    new_steward_wallet, steward_did = sdk_add_new_steward(looper,
+                                                          sdk_pool_handle,
+                                                          sdk_wallet_steward,
+                                                          'another_ste')
 
     # Send another pool ledger request (NODE) but don't wait for completion of
     # request
     next_node_name = 'next_node'
-    r = sendAddNewNode(tdir, tconf, next_node_name, new_steward, new_steward_wallet)
-    node_req = r[0]
+
+    node_req = looper.loop.run_until_complete(
+        build_request(tconf, tdir, next_node_name, steward_did))
+    sdk_wallet = (new_steward_wallet, steward_did)
+    signed_reqs = sdk_sign_request_objects(looper, sdk_wallet,
+                                           [sdk_json_to_request_object(
+                                               json.loads(node_req))])
+    request_couple = sdk_send_signed_requests(sdk_pool_handle, signed_reqs)[0]
 
     # Send more domain ledger requests but don't wait for replies
-    requests = [node_req, *
-                sendRandomRequests(new_steward_wallet, new_steward, 5)]
+    request_couples = [request_couple, *
+    sdk_send_random_requests(looper, sdk_pool_handle, sdk_wallet_client, 5)]
 
     # Make sure all requests are completed
-    waitForSufficientRepliesForRequests(looper, new_steward,
-                                        requests=requests)
+    total_timeout = sdk_eval_timeout(len(request_couples), len(txnPoolNodeSet))
+    sdk_get_and_check_replies(looper, request_couples)
 
     # Make sure pool is functional
-    ensure_pool_functional(looper, txnPoolNodeSet, wallet1, client1)
+    sdk_ensure_pool_functional(looper, txnPoolNodeSet,
+                               sdk_wallet_client, sdk_pool_handle)
+
+
+async def build_request(tconf, tdir, next_node_name, steward_did):
```

Comment:
It can construct only NODE request, so it should be renamed to sdk_build_node_request